docs/dymos_book/getting_started/transcriptions.md

```diff
@@ -0,0 +1,102 @@
+
+To solve these problems directly using nonlinear optimizers, we have to find a way to discretize this continuous problem into a form that can be solved by a nonlinear optimizer. This process is called _transcription_.
+
+Dymos supports two forms of direct transcription: [collocation](getting_started:collocation:what_is_collocation) and explicit shooting.
+
+Collocation-based optimal control methods, also known as pseudospectral methods, are implicit techniques.
+The entire state and control history of the system is proposed at some discrete set of points in time.
+We can then calculate how accurately the proposed state and control history obeyed the ODE governing the system dynamics - this is a residual we call the _defects_.
+By iteratively varying the states, controls, and elapsed time of the trajectory we can reduce the defects to zero, meaning that the proposed state history is a solution to the ODE of the system, with the given control history and time of propagation.
+
+In explicit shooting, an explicit numerical integration technique is used to propagate the initial state over the given time duration subject to the given controls. The process of integrating the trajectory itself eliminates the defects.
+
+## Differences between collocation and explicit shooting
+
+### Design Variables
+
+The design variables for an explicit shooting problem include:
+- the initial state (assuming it's not fixed, and assuming a single integration interval (_single shooting_))
+- the initial state in each integraiton interval (for _multiple shooting_)
+- the control values at some set of discrete points in time
+- the elapsed time of the trajectory.
+
+The design variables for a collocation problem include:
+- the states at some set of points along the trajectory (the initial and final ones may not be included if they're fixed)
+- the controls at discrete points in time
+- the elapsed time of the trajectory.
+
+The design variable vector can be considerably larger for the implicit collocation techniques, depending on the size of the state vector and the number of discrete points in time.
+
+### Constraints
+
+For implicit collocation techqniques the constraints include:
+- the defect constraints for each state at various discrete points in time throughout the trajectory
+- continuity constraints that ensure the state values remain continuous (for _multiple shooting_)
+- continuity constraints on the control values and (optionally) their rates
+- any other path or boundary constraints imposed
+
+For explicit shooting techqniques the constraints include:
+- continuity constraints that ensure the state values remain continuous (assuming multiple integration intervals (_multiple shooting_))
+- continuity constraints on the control values and (optionally) their rates
+- any other path or boundary constraints imposed
+
```

Comment:
Under design variables, you do Shooting first, then Collocation.  Here, you do Collocation first, then Shooting. I would pick one order and be consistent.